app/src/lib/api.ts

```diff
@@ -104,6 +104,7 @@ export interface IAPIRepository {
   readonly default_branch: string
   readonly pushed_at: string
   readonly parent?: IAPIRepository
+  readonly has_issues?: boolean
```

Comment:
I was under the impression that `has_issues` is a mandatory field in API responses except for a few rare cases which we shouldn't have to concern ourselves with (public repository listing, checks, notifications etc). Do we know of an endpoint that we rely on to upsert GitHubRepositories which doesn't provide `has_issues`?

Reply:
ah, you are right! that was my mistake.
```suggestion
  readonly has_issues: boolean
```